### PR TITLE
Showed ip in logs with Camera,CR create/update and Archive create/delete.

### PR DESCRIPTION
**Pull request by affan00**
@azharmalik3

### CODE REVIEW

File: app/assets/javascripts/cameras/single/logs.js.coffee [hunk 1]
```diff
@@ -71,9 +71,18 @@ initializeDataTable = ->
       {data: ( row, type, set, meta ) ->
         if row.action is 'shared' or row.action is 'stopped sharing'
           if row.extra && row.extra.with
-            return row.action + ' with ' + (row.extra.with if row.extra)
+            return row.action + ' with ' + (
+              row.extra.with + "," + " ip: " + row.extra.ip  if row.extra
```
Comment (azharmalik3) [inline app/assets/javascripts/cameras/single/logs.js.coffee]:
```
return row.action + ' with ' + row.extra.with +  ( ", ip: #{row.extra.ip}"  if row.extra.ip)
```

File: app/assets/javascripts/cameras/single/logs.js.coffee [hunk 2]
```diff
@@ -71,9 +71,18 @@ initializeDataTable = ->
...
+            )
           else
             return row.action
+        if row.action is 'edited' or
+          row.action is 'created' or
+          row.action is 'cloud recordings updated' or
+          row.action is 'cloud recordings created' or
+          row.action is 'archive created' or
+          row.action is 'archive deleted'
+            return row.action + ',' + ' ip: ' + (row.extra.ip if row.extra)
```
Comment (azharmalik3) [inline app/assets/javascripts/cameras/single/logs.js.coffee]: ```return row.action + (", ip: #{row.extra.ip}" if row.extra.ip)```

**Comment by azharmalik3** [comment]
:+1:

File: app/assets/javascripts/cameras/single/logs.js.coffee [hunk 3]
```diff
@@ -69,11 +69,21 @@ initializeDataTable = ->
...
+        ip = ""
+        if row.extra and row.extra.ip
+          ip = ", ip: #{row.extra.ip}"
         if row.action is 'shared' or row.action is 'stopped sharing'
           if row.extra && row.extra.with
-            return row.action + ' with ' + (row.extra.with if row.extra)
+            return ("#{row.action} with #{row.extra.with}") + ip
           else
             return row.action
+        if row.action is 'edited' or
```
Comment (naveed-ahmad) [inline app/assets/javascripts/cameras/single/logs.js.coffee]: how about 

```coffee
if row.action in ['created', 'edited', 'cloud recordings updated', 'cloud recordings created', 'archive created', and what not]
  row.action + ip
else ...
```

Better yet create constants for these events, sorry I'm not sure what's going on here but there is always good way and "not that good way" to do the same thing.

**Comment by marcoherbst** [comment]
:+1 Thanks @naveed-ahmad

**Pull request merged**